serverless/functions/how-to/package-function-dependencies-in-zip.mdx

```diff
@@ -18,49 +18,55 @@ This feature can help you to add your libraries or static files to your function
 
 <Macro id="requirements" />
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
-  - You have [created a functions namespace](/serverless/functions/how-to/create-a-functions-namespace/)
-  - You know how to [create a Function](/serverless/functions/how-to/create-a-function/)
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- A [Functions namespace](/serverless/functions/how-to/create-a-functions-namespace/)
+- Read the [how to create a Function](/serverless/functions/how-to/create-a-function/) page
```

Comment:
@SamyOubouaziz The last one, I don't know. It must be aligned with the intro 'You must have', so I wrote 'Read the page'. wdyt?

Reply:
I'll remove this part as this page explains how to create a function